packages/app/src/state/futures/selectors.ts

```diff
@@ -920,6 +920,7 @@ export const selectEditPositionPreview = createSelector(
 	selectEditPositionInputs,
 	(state: RootState) => state.futures,
 	(type, { nativeSizeDelta }, futures) => {
+		if (isNaN(Number(nativeSizeDelta))) return null
```

Comment:
I'm not sure if this is the best place to fix, there shouldn't be a preview when there is no position size so this is more a workaround and introduces some different logic to the other preview selectors.